go/vt/vttablet/tabletmanager/rpc_external_reparent.go

```diff
@@ -221,22 +250,24 @@ func (agent *ActionAgent) finalizeTabletExternallyReparented(ctx context.Context
 			errs.RecordError(err)
 		}
 	}()
-	if !topoproto.TabletAliasIsZero(oldMasterAlias) {
+	wg.Wait()
```

Comment:
Do we need a new Wait here? These steps were previously done concurrently on purpose, since they aren't interdependent.